migrations/update_filtered_positives.py

```diff
@@ -0,0 +1,101 @@
+from crawler.helpers import get_config
+from datetime import datetime
+from migrations.helpers.update_filtered_positives_helper import (
+    pending_plate_barcodes_from_dart,
+    positive_result_samples_from_mongo,
+    update_filtered_positive_fields,
+    update_mongo_filtered_positive_fields,
+    update_mlwh_filtered_positive_fields,
+    update_dart_filtered_positive_fields,
+)
+from migrations.helpers.shared_helper import print_exception
+from crawler.filtered_positive_identifier import FilteredPositiveIdentifier
+
+
+def run(settings_module: str = "") -> None:
+    """Updates filtered positive values for all positive samples in pending plates
+
+    Arguments:
+        config {ModuleType} -- application config specifying database details
+    """
+    print("-" * 80)
```

Comment:
Same about logging here but might mean moving a few modules around to get to the logging config.